devicon.json

```diff
@@ -7417,6 +7417,30 @@
         "color": "#130754",
         "aliases": []
     },
+    {
+        "name": "passport",
+        "altnames": [
+            "passportjs"
+        ],
+        "tags": [
+            "authentication",
+            "security"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#35df79",
```

Comment:
Because the color `#35df79` was chosen, all the font versions of the icon, in this case `plain` and `plain-wordmark`, must have this as their path fill color.